### DISCUSSION TITLE
how to load another html page before page is mounted

### DISCUSSION THREAD

**Discussion by BilalKhatib** [vuejs/petite-vue#191] [Q&A]
is there a way to load an html page before my page is mounted ? 

example : i have 2 pages Page1.html and Page2.html

my createApp is inside Page2.html
how can i load my Page1.html with it's scripts inside before my Page2.html mount ?

**Comment by sqllyw** [comment]
Hi,

the doc says:

it is specifically optimized for "sprinkling" a small amount of interactions on an existing HTML page rendered by a server framework

I'll assume your app is a multi page app, then page2.html checks certain condition before mounting, if not met, you force page1.html to be loaded, if that's the case, then you can use vue:mounted event to check, and location.href = 'page1.html' to load page1

or even better:

check the condition before createApp, location.href='page1.html' if needed, no need for the even mounted
